Patches/Antinium/ThingDefs_Races/Ant_AntiniumRace.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Antinium (Continued)</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<!-- Antinium alien race (Ant_AntiniumRace) -->
+				<li Class="PatchOperationAddModExtension">
+					<xpath>Defs/AlienRace.ThingDef_AlienRace[defName="Ant_AntiniumRace"]</xpath>
+					<value>
+						<li Class="CombatExtended.RacePropertiesExtensionCE">
+							<bodyShape>Humanoid</bodyShape>
+						</li>
+					</value>
+				</li>
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/AlienRace.ThingDef_AlienRace[defName="Ant_AntiniumRace"]/comps</xpath>
+					<value>
+						<li>
+							<compClass>CombatExtended.CompPawnGizmo</compClass>
+						</li>
+						<li Class="CombatExtended.CompProperties_Suppressable" />
+					</value>
+				</li>
+				<!-- Stat bases  -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/AlienRace.ThingDef_AlienRace[defName="Ant_AntiniumRace"]/statBases/ShootingAccuracyPawn</xpath>
+					<value>
+						<AimingAccuracy>0.8</AimingAccuracy>
+						<MeleeCritChance>1.17</MeleeCritChance>
+						<MeleeParryChance>1.05</MeleeParryChance>
+						<ReloadSpeed>1.5</ReloadSpeed> <!-- Extra hands means extra speed when reloading -->
+						<CarryWeight>60</CarryWeight> <!-- Ants can lift more than they weight irl -->
+						<Mass>72</Mass> <!-- A human weighs 60 kg, but ants have chitin so mass is increased -->
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/AlienRace.ThingDef_AlienRace[defName="Ant_AntiniumRace"]/statBases/MeleeDodgeChance</xpath>
+					<value>
+						<MeleeDodgeChance>0.9</MeleeDodgeChance>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/AlienRace.ThingDef_AlienRace[defName="Ant_AntiniumRace"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<ArmorRating_Sharp>1</ArmorRating_Sharp>
```

Comment:
I'd reduce this to around 0.25mm or so, as 1mm is going to make them completely protected against most sharp attacks from animals and many basic melee weapons. Around 0.25mm would provide a decent damage reduction without completely stopping some attacks.

Reply:
This is in line with the intent of the Antinium mod. Antinium are a race from the novel The Wandering Inn by pirateaba. Oft compared to an unkillable, unending army of cockroaches, they are monikered "The Black Tide."

Reply:
I guess I could either decrease the sharp armor rating, or increase the combat power of the pawn kinds. I'm guessing to decrease it to 0.33mm RHA with 1 MPa since there's already an offset to the pawn combat power (when comparing to tribals).

Reply:
I guess I could either decrease the sharp armor rating, or increase the combat power of the pawn kinds. I'm guessing to decrease it to 0.33mm RHA with 1 MPa since there's already an offset to the pawn combat power (when comparing to tribals).